uint8_t target_blue_val
---
uint32_t calibrated_exposure
string message
int8_t k4aExposureServiceErrorCode